activesupport/lib/active_support/claims.rb

```diff
@@ -0,0 +1,63 @@
+require 'active_support/time'
+
+module ActiveSupport
+  # +Claims+ supports <tt>ActiveSupport::MessageVerifier</tt> by
+  # representation of message in the form JWT Claims and verifying
+  # the claims of the signed message. 
```

Comment:
This documentation is extremely meager and doesn't really make sense to me. If you can't come up with anything better, let's cut it.